client/my-sites/activity/activity-log-confirm-dialog/index.jsx

```diff
@@ -49,7 +49,7 @@ const ActivityLogConfirmDialog = ( {
 			<div className="activity-log-confirm-dialog__partial-restore-settings">
 				<p>
 					{ notice
-						? translate( 'Choose the items you wish to rewind:' )
+						? translate( 'Choose the items you wish to restore:' )
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.